### DISCUSSION TITLE
[Unstable] Cannot use item orders on rum, ethanol, or cigars

### DISCUSSION THREAD

**Discussion by ReesesCups** [FakeFishGames/Barotrauma#13666] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

You cannot use orders on rum, ethanol, or cigars, such as telling npcs to not interact. This may also be a problem for other items that recently received changes. The cigar sprite is different, and ethanol and rum received the ability to fight infections, so I believe there may have been an error created there. 

### Reproduction steps

1. Make a bottle of rum using raptorbane and ethanol (or cheat it into the game; you can't use additional cargo for rum for some reason).
2. Try to tell your profoundly stupid npc medics to not use your delicious alcoholic beverage by shift-middle mouse clicking on the bottle in a cabinet of your choice
3. cry as they refuse to hear your pleas
4. cry harder as the same applies as well to ethanol and cigars

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.2.13.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by ReesesCups** [comment]
small note - for cigars, the AI cannot use them, but perhaps it would still be worthwhile to allow orders for it, such as for non-bot crew. Same goes for pipe tobacco. If the AI has terrible psychosis, they still won't use it. Ethanol and Rum will be used by the AI to fight infections and you can't order them not to use it. Given that this is not the best use of ethanol and definitely not the best use of rum, I believe these items definitely need orders.

**Comment by HawkinsThe71st** [comment]
It might be because those items are not deconstructible as other non-deconstructible items such as materials also cannot be shift + middle mouse clicked inside containers (if dropped on the ground they can be shift + middle mouse clicked to give the ignore order)

![image](https://github.com/FakeFishGames/Barotrauma/assets/115734437/3f956fda-fb17-45be-af52-d2720a645ba4)

**Reply by Regalis11** [reply]
Yep, this seems to be the cause: the contextual orders don't seem to show up on any item that's not deconstructible. We'll look into fixing it!

**Comment by Regalis11** [comment]
This has been addressed now in our private repo, fix coming up in the next unstable build.
